### PR TITLE
chore: Bump React Native to v0.70

### PR DESCRIPTION
**Pull request by tuliomir**
### Acceptance Criteria
- Bumps the React Native to v0.70
- Switches the Camera lib from `react-native-vision` to [`react-native-camera-kit`](https://github.com/teslamotors/react-native-camera-kit), as it was causing compatibility issues with the rn
- Various visual color fixes

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Review by r4mmer — APPROVED** [review]

File: patches/react-native+0.69.10.patch
Comment (andreabadesso) [inline patches/react-native+0.69.10.patch:1]: Remove?

File: src/App.js [hunk 1]
```diff
@@ -562,6 +562,11 @@ const RootStack = () => {
         headerShown: false,
         gestureEnabled: false,
       }}
+      theme={{
+        colors: {
+          background: 'red'
+        }
+      }}
```
Comment (andreabadesso) [inline src/App.js:569]: Remove?

**Review by andreabadesso — CHANGES_REQUESTED** [review]

File: patches/react-native+0.69.10.patch
Comment (tuliomir) [reply patches/react-native+0.69.10.patch:1]: Done on [99f21e0](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/295/commits/99f21e0d9df25d989a13123abae4cb446d6c783a)

File: src/App.js (hunk 1 above)
Comment (tuliomir) [reply src/App.js]: Done on [99f21e0](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/295/commits/99f21e0d9df25d989a13123abae4cb446d6c783a)

**Review by andreabadesso — APPROVED** [review]

**Pull request merged**